soc/sensry/ganymed/Kconfig

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 sensry.io
+
+config SOC_FAMILY_GANYMED
+	select BUILD_OUTPUT_HEX
+	select BUILD_OUTPUT_BIN
```

Comment:
why are both of these always required?

Reply:
actually really need only binary. removed the hex options